### PR TITLE
Make some changes to home and about pages.

### PR DESCRIPTION
**Pull request by Mo7amedSMahdi**
- Add mobile menu slide-in animation.
- Change the title font to cocogoose on the home and about page.
- Change the hero background image.
- Change the title background image to have a red image effect on the title for the home page.
- Implement the mobile menu animation in javascript.
- Add keyframe animation for the mobile menu in main.css.
- Add a border on hover for the program cards.
-
https://www.loom.com/share/a2f19a813b7b4566b7852051bf3f18b1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,46 +5,56 @@
...
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
+        <div id="mobileNav" class="hidden">
+            <button id="btnClose" type="button" class="btn btn-icon close hidden" data-target="mobileMenuClose"><i
+                    class="icon-close"></i></button>
+            <ul id="mobileNavMenu" class="nav--mobile">
+                <li class="active"><a href="index.html">Home</a></li>
+                <li><a href="about.html">About</a></li>
+            </ul>
+        </div>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please add this bottom to maintain the main [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
Addbutton

File: about.html [hunk 2]
```diff
@@ -5,10 +5,123 @@
...
+
+    <footer class="footer flex">
+        <div class="footer__group">
+            <img src="images/brands/it_live.png" alt="creative commones korea logo">
+        </div>
+        <div class="footer__group span-2">
+            <p>"Like Never Before"</p>
+            <p>2019 The largest Iraqi forum for information technology in all its disciplines.</p>
+        </div>
+    </footer>
```
Comment (CollinsTatang) [inline about.html:123]: - [x] Please the about footer page should be center as the home page footer
cap-footers

File: about.html [hunk 3]
```diff
@@ -5,10 +5,123 @@
...
+            <a href="#"><img src="images/icons/facebook.svg" alt="facebook" /></a>
+            <a href="#"><img src="images/icons/twitter.svg" alt="twitter" /></a>
+            <div class="button-group flex">
+                <button type="button" class="btn btn-text">English</button>
+                <button type="button" class="btn btn-text">My Page</button>
+                <button type="button" class="btn btn-text">Logout</button>
+            </div>
+        </div>
+        <div id="mobileNav" class="hidden">
+            <button id="btnClose" type="button" class="btn btn-icon close hidden" data-target="mobileMenuClose"><i
```
Comment (CollinsTatang) [inline about.html]: - [x] Kindly fix the closing button it doesn't work

https://user-images.githubusercontent.com/55596970/158561315-4f83d24f-a0c6-4a9a-8efb-41d83e5249a6.mp4

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Mo7amedSMahdi , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 1 above)
Comment (ShoiraTa) [reply index.html]: Adressed 💪🏻

File: about.html (hunk 2 above)
Comment (ShoiraTa) [reply about.html:123]: Adressed 👍🏻

File: about.html (hunk 3 above)
Comment (ShoiraTa) [reply about.html]: Adressed 👍🏻

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @Mo7amedSMahdi , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**